streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -192,7 +192,6 @@ public void resume() {
             }
             transactionInFlight = true;
         }
-        initTopology();
```

Comment:
We `initTopology()` already in `AssignedTasks#maybeResumeSuspendedTask()` L252 within `transitionToRunning()`

Reply:
Good catch!